dev/com.ibm.ws.opentracing_fat/publish/servers/OpentracingTCKServer/server.xml

```diff
@@ -0,0 +1,28 @@
+<!--Copyright (c) 2017 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>webProfile-7.0</feature>
+        <feature>localConnector-1.0</feature>
+        <feature>cdi-1.2</feature>
+	<feature>opentracing-1.0</feature>
+	<feature>mpOpenTracing-1.0</feature>
+	<feature>usr:opentracingMock-0.30</feature>
```

Comment:
I do not know enough to check that this unusual <feature>usr:opentracingMock-0.30</feature>
is not a locla unit testing feature that only lives on a developers laptop? Or is fine. :-)

Reply:
Thanks Gordon for the review! I removed blank-tck-suite.xml. The user feature is expected as it has the Tracer under test, and it's built by the FAT itself (personal build passed). Thanks.